src/components/AvgMonthPaymentWidget/avg_month_payment_widget.latte

```diff
@@ -0,0 +1 @@
+<li n:ifset="$avgMonthPayment" class="list-group-item"><b>{_segment.fields.avg_month_payment}:</b> {$avgMonthPayment|price}</li>
```

Comment:
Translation string should now be loaded from payments module translations and removed from segments module translations.